src/components/DisabledFormOverlay/DisabledFormOverlay.less

```diff
@@ -16,7 +16,7 @@
 .multisig_ban_container {
     font-size: 16px !important;
     .is_multisig {
-        width: 100%;
+        width: 12rem;
```

Comment:
we shouldn't be fixing the overlay issue by setting a fixed width imo. Do you think there might be another way to  fix this while keeping the width as 100% ?